packages/blade/src/components/Indicator/__tests__/Indicator.native.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { Indicator } from '..';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.native';
+
+describe('<Indicator />', () => {
+  it('should render', () => {
+    const { toJSON } = renderWithTheme(<Indicator accessibilityLabel="Success" />);
+
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render different variants for size and intent', () => {
+    const { toJSON } = renderWithTheme(
+      <Indicator accessibilityLabel="Warning" size="small" intent="notice" />,
+    );
+
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render text label', () => {
+    const label = 'Active';
+    const { getByText } = renderWithTheme(<Indicator>{label}</Indicator>);
+
+    getByText(label);
```

Comment:
shouldn't we assert here rather than just calling this function?

Reply:
This automatically fails the test if the element isn't found, so we don't need to explicitly assert the element after `getBy*`